Fix typos in `cell_diffusion_timescale` methods.

ali-ramadhan opened the pull request.

glwagner commented:
This PR might want to add a simple test that runs these functions to ensure this doesn't continue to happen (and improve coverage), rather than only fixing the current code.

ali-ramadhan commented:
@glwagner I added turbulence diagnostics tests to ensure that `DiffusiveCFL` is compatible with all closures.

Unfortunately it failed on `TwoDimensionalLeith` as it was bundled with `AbstractSmagorinsky` and `TwoDimensionalLeith` does not have a `closure.Pr` field.

I split it off into its own `cell_diffusion_timescale` where it's easy to calculate the viscous CFL condition but it seems that `TwoDimensionalLeith` calculates the nonlinear diffusivity on the fly and so there is no easy/cheap way to compute the diffusive CFL for each tracer.

I'm thinking of merging this PR because I need the bugfix for a script in another repo.

I will open an issue about the diffusive CFL for `TwoDimensionalLeith` (out of scope for this PR).
